### DISCUSSION TITLE
[Cloud Engineer] Scale infrastructure with user load

### DISCUSSION THREAD

**Discussion by jawache** [Green-Software-Foundation/patterns#31] [Ideas]
https://docs.aws.amazon.com/wellarchitected/latest/sustainability-pillar/scale-infrastructure-with-user-load.html

Intent: Identify periods of low or no utilization and scale resources to eliminate excess capacity and improve efficiency.

**Comment by holanita** [comment]
[Issue created](https://github.com/Green-Software-Foundation/green-software-patterns/issues/71)
